commands/handbook.py

```diff
@@ -0,0 +1,72 @@
+from commands.base import Command
+from discord import Embed
+from helpers import CommandFailure, is_good_response
+
+import bs4, re, requests
+
+
+def subject_details(code):
+    code = code.upper()
+
+    page = requests.get('https://www.handbook.unsw.edu.au/undergraduate/courses/2019/' + code)
+
+    if not is_good_response(page):
+        page = requests.get('https://www.handbook.unsw.edu.au/postgraduate/courses/2019/' + code)
+
+        if not is_good_response(page):
+            return None
+
+    soup = bs4.BeautifulSoup(page.text, features='lxml')
+
+    course_title = soup.find('span', attrs={'data-hbui' : 'module-title'}).string.strip()
+    course_offerings = soup.find('strong', string=re.compile('Offering Terms')).parent.contents[3].string.strip()
+
+    course_conditions_tag = soup.find('div', id='readMoreSubjectConditions')
+
+    course_conditions = ''
```

Comment:
You can just make `course_conditions = 'None'` by default and get rid of the `else`

Reply:
+=